api/src/pcapi/alembic/versions/20230721T065239_48013aa5a395_remove_ff_allow_idcheck_registration_for_educonnect_eligible.py

```diff
@@ -0,0 +1,36 @@
+"""
+remove_ff_ALLOW_IDCHECK_REGISTRATION_FOR_EDUCONNECT_ELIGIBLE
```

Comment:
(détail) En plus lisible (comme le suggère le commentaire généré par défaut par Alembic) : 

```suggestion
Remove ALLOW_IDCHECK_REGISTRATION_FOR_EDUCONNECT_ELIGIBLE feature flag
```